Backend/src/lib/firebase.ts

```diff
@@ -1,6 +1,5 @@
 import admin from 'firebase-admin';
 import appConfig from '@/AppConfig/app.config';
-
```

Comment:
nit: always separate imports from the rest of the code with at least one empty line.